src/packagedef.jl

```diff
@@ -116,6 +116,15 @@ function set_compiled_methods()
     push!(compiled_modules, Base.Threads)
 end
 
+function _have_fma_compiled(T)
+    Base.Experimental.@force_compile
+    Core.Intrinsics.have_fma(T)
+end
```

Comment:
I think you want to explicitly force specialization here, I don't think `@force_compile` does much:
```suggestion
_have_fma_compiled(::Type{T}) where {T} = Core.Intrinsics.have_fma(T)
```